oeps/processes/oep-0014-proc-archive-repos.rst

```diff
@@ -88,6 +88,14 @@ These steps should be followed for all repos within the Open edX organization (f
 
   - Move the repository to the openedx-unsupported organization
 
+  - Remove the repo from ``openedx-translations`` and Transifex
+    
+    - Remove the repo entry from the ``extract-translation-source-files`` workflow
+    
+    - Remove the repo directory from the ``translations`` directory
+
+    - Remove the repo resource from the ``openedx-translations`` project on Transifex
```

Comment:
Would it be better to say "ask a Transifex admin to..."? I don't think most people will be able to do this

Reply:
I think an axim-engineering request. And maybe as part of this PR, we do an audit of admins on Transifex and remove people who aren't around or who don't need access and add a couple more Axim folx.